datacats/userprofile.py

```diff
@@ -131,7 +131,6 @@ def create(self, environment, target_name, stream_output=None):
         remote_server_command(
             ["ssh", environment.deploy_target, "create", target_name],
             environment, self,
-            stream_output=stream_output,
```

Comment:
it's dead code, it does nothing but it looks like it might do something, likely leading to user confusion. Why would you keep it? Should probably be getting flagged by our linter too